plugin/storage/es/spanstore/writer.go

```diff
@@ -132,7 +132,7 @@ func getSpanAndServiceIndexFn(archive, useReadWriteAliases bool, prefix string)
 }
 
 // WriteSpan writes a span and its corresponding service:operation in ElasticSearch
-func (s *SpanWriter) WriteSpan(span *model.Span) error {
+func (s *SpanWriter) WriteSpan(ctx context.Context, span *model.Span) error {
```

Comment:
Can we mark this as `_ context.Context` if it is not being used right now? 

Also, `SpanWriter` has a `context` field which is probably redundant if `WriteSpan` can be called concurrently, IIRC it can.

Reply:
+1